elcometer456_block.py

```diff
@@ -42,13 +58,15 @@ def _read_gage(self):
             self.logger.debug('Waiting for reading')
             try:
                 raw = self._serial.readline()
-            except:
+            except serial.SerialException:
                 if not self._stopping:
-                    self.logger.warning('serial failed read', exc_info=True)
+                    self.logger.info('Closing serial connection')
+                    self._serial.close()
                 continue
             self._serial.write(b"O")
-            read = b''
+            read = 0
```

Comment:
But do we want to initialize it as `None` or `0` given that this is going to be used on the front end?